src/omnisharp/protocol.ts

```diff
@@ -477,8 +476,10 @@ export namespace V2 {
         export const GetTestStartInfo = '/v2/getteststartinfo';
```

Comment:
This command can probably be removed as well, as it was used by the `project.json` support. Let me know if you'd like me to go through and remove it as well.

Reply:
If you are interested, you could remove in a follow up PR or open a tracking issue with the suggestion. Thanks!